manifests/a/aandrew-me/ytDownloader/3.15.0/aandrew-me.ytDownloader.installer.yaml

```diff
@@ -0,0 +1,26 @@
+# Created with Komac v1.6.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: aandrew-me.ytDownloader
+PackageVersion: 3.15.0
+InstallerLocale: en-US
+InstallerType: msi
+Scope: user
```

Comment:
```suggestion
```
This can install as machine scope if run as admin

Reply:
Done.